Development/UnitTesting/Sandboxing/Sandbox.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+// 
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Security;
+using System.Security.Permissions;
+using System.Security.Policy;
+using Remotion.Utilities;
+
+// Note: This file is originally defined in Remotion.Development.UnitTesting.Sandboxing. It is duplicated by Remotion.Linq.UnitTests.Sandboxing.
+// Note: Changes made to this file must be synchronized with all copies.
```

Comment:
Should this be integrated into the other projects at some point?

Reply:
Since we're going to fork the whole sandboxing infrastucture into each git repostory, yes, the file should be intgrated into the respective fork.

Reply:
Sorry I meant something else. Should the change I made to these files (see below) be integrated into the remotion & relinq code?

Reply:
Haven't yet looked at the changes. it's definitely out of scope for the current task. The feautre will be dropped in re-motion, I think and then we only need it in re-linq and maybe just create a ticket for this including the motivation and let's see what comes of it?